README.md

```diff
@@ -32,3 +32,116 @@ zopen build
 Note that running `zopen build` against your development repo will NOT install the code into your
 environment.
 
+
+# Developer Notes
+
+## workflow
```

Comment:
Can you capitalize the first letter in the headers?

Reply:
Yes that would be wise